File: app/front/src/components/main/dto/add_xml/updateWkTblAddByXmlResultDto.ts
import { FrameworkMessageAndResultDto, type FrameworkMessageAndResultDtoInterface } from "seijishikin-jp-normalize_common-tool";
import { WkTblMasterAllByXmlEntity, type WkTblMasterAllByXmlEntityInterface } from "../../entity/wkTblMasterAllByXmlEntity";

interface UpdateWkTblAddByXmlResultDtoInterface extends FrameworkMessageAndResultDtoInterface {

    /** 編集対象Entity */
    wkTblMasterAllByXmlEntity: WkTblMasterAllByXmlEntityInterface;

}

/**
 * 企業／団体一括登録履歴ワークテーブル更新CapsuleDto
 */
class UpdateWkTblAddByXmlResultDto extends FrameworkMessageAndResultDto
    implements UpdateWkTblAddByXmlResultDtoInterface {

    /** 編集対象Entity */
    wkTblMasterAllByXmlEntity: WkTblMasterAllByXmlEntityInterface;

    constructor() {
        super();
        this.wkTblMasterAllByXmlEntity = new WkTblMasterAllByXmlEntity()
    }
}

export { type UpdateWkTblAddByXmlResultDtoInterface, UpdateWkTblAddByXmlResultDto }

File: app/back/src/main/java/net/seijishikin/jp/normalize/manage/kanrensha/dto/add_xml/UpdateWkTblAddByXmlResultDto.java
package net.seijishikin.jp.normalize.manage.kanrensha.dto.add_xml;

import java.io.Serializable;

import net.seijishikin.jp.normalize.common_tool.dto.FrameworkMessageAndResultDto;
import net.seijishikin.jp.normalize.manage.kanrensha.entity.WkTblMasterAllByXmlEntity;


/**
 * 企業／団体一括登録履歴ワークテーブル更新CapsuleDto
 */
public class UpdateWkTblAddByXmlResultDto extends FrameworkMessageAndResultDto // NOPMD DataClass
        implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 編集対象Entity */
    private WkTblMasterAllByXmlEntity wkTblMasterAllByXmlEntity = new WkTblMasterAllByXmlEntity();

    /**
     * 編集対象Entityを取得する
     *
     * @return 編集対象Entity
     */
    public WkTblMasterAllByXmlEntity getWkTblMasterAllByXmlEntity() {
        return wkTblMasterAllByXmlEntity;
    }

    /**
     * 編集対象Entityを設定する
     *
     * @param wkTblMasterAllByXmlEntity 編集対象Entity
     */
    public void setWkTblMasterAllByXmlEntity(final WkTblMasterAllByXmlEntity wkTblMasterAllByXmlEntity) {
        this.wkTblMasterAllByXmlEntity = wkTblMasterAllByXmlEntity;
    }

}
